system_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -exo pipefail
+
+source ./helpers.sh
+
+remove_containers
+
+case "$1" in
+    "sqlite3" )
+    rm -fr /tmp/fn_system_tests.db
+    touch /tmp/fn_system_tests.db
+    export FN_DB_URL="sqlite3:///tmp/fn_system_tests.db"
+    ;;
+
+    "mysql" )
+    DB_CONTAINER="func-mysql-test"
+    docker rm -fv ${DB_CONTAINER} || echo No prev mysql test db container
+    docker run --name ${DB_CONTAINER} -p 3306:3306 -e MYSQL_DATABASE=funcs -e MYSQL_ROOT_PASSWORD=root -d mysql
+    MYSQL_HOST=`host ${DB_CONTAINER}`
+    MYSQL_PORT=3306
```

Comment:
also the ports collide with api_test.sh, not a super important issue, but we do run the builds in parallel.

Reply:
good spot, we changed the port and the name of the containers